data/kernels/colorspace.cl

```diff
@@ -325,3 +325,37 @@ inline float4 sRGB_to_XYZ(float4 sRGB)
   return XYZ;
 }
 
+inline float4 XYZ_to_JzAzBz(float4 XYZ_D65)
+{
+  float4 temp1, temp2;
```

Comment:
you can store the constants into `float4` variables and unroll the dot products like `temp2.x = cnst.x * XYZ.x + cnst.y * XYZ.y + cnst.z * XYZ.z`. Since some GPU have an hardware acceleration for dot products, it could give a hint to the compiler that it can be used here.

Reply:
Done for the M and A matrices